docs/configuration/features/text_features.md

```diff
@@ -369,7 +371,7 @@ Text output features can be used for either tagging (classifying each token of a
 generation (generating text by repeatedly sampling from the model). There are two decoders available for these tasks
 named `tagger` and `generator` respectively.
 
-Example text feature entry using a tagger decoder (with default parameters) in the output features list:
+Example sequence output feature using default parameters:
```

Comment:
I would actually revert back to text feature in the description because this might be confusing to users otherwise (they might think it's a typo although it logically makes sense). Wdyt?

Reply:
Typo, will fix.